stylesheet.css

```diff
@@ -1,195 +1,742 @@
-html {     
-  box-sizing: border-box; 
-} 
-
 *, *:before, *:after {     
-  box-sizing: inherit; 
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-body {
-  background-color: white;
-  font-family: Verdana, Arial, sans-serif;
-  margin-left: 25%;
-  margin-right: 25%;
+ul {
+  list-style-type: none;
 }
 
 li {
-  display: inline; 
+  text-decoration: none;
 }
-
 /*-- link --*/
+a {
+  text-decoration: none;
+  color: initial;
+}
+
 a:link {
   text-decoration: none;
   color: black;
 }
 
 /*-- link hovered --*/
 a:hover {
-    background-color: blue;
-    color: white;
+  background-color: blue;
+  color: white;
 }
 
-/*-- sections header --*/
-.section-header {
+/*body*/
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  width: 100%;
+  margin: auto;
+  background-position: 0 5rem;
+  background: url("images/header.png") no-repeat;
+  box-sizing: border-box;
+  line-height: 1;
+}
+
+body section {
+  padding: 0 1.25rem;
+}
+
+body > header {
+  padding: 0 calc((100vw - 52rem)/2);
+  width: 100%;
+  background-color: white;
+  margin: auto;
+  height: 10vh;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  font-family: "Montserrat";
+  z-index: 1;
+}
+
+body > header > img {
+  padding-left: 1rem;
+}
+
+body > header > nav > ul {
+  display: none;
+}
+
+/* main body footer*/
+body > footer {
+  background: url(images/footer-mobile.png) no-repeat;
+  background-size: cover;
+  font-size: 0.8rem;
+  line-height: 1.4;
+  color: white;
+  text-align: center;
+  font-weight: 100;
+  padding: 2rem 0;
+}
+
+/*-- section's header --*/
+body > section > header {
+  font-size: 2.25rem;
   text-align: center;
-  font-size: 40px;
-  font-weight: 900;
+  text-transform: uppercase;
+  position: relative;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  padding: 4rem 0 0;
 }
 
-/*-- img in sections --*/
-.img-sec {
-  display: -webkit-flex;
-  display: -ms-flexbox;
+/* section intro*/
+#intro {
   display: flex;
-  -webkit-justify-content: center;
-  -ms-flex-pack: center;
-  justify-content: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
+  flex-flow: column wrap;
+  padding-top: 5rem;
+  color: white;
+  font-family: "Montserrat";
+  padding-bottom: 3rem;
+}
+
+#intro header {
+  text-align: left;
+}
+
+#intro h1 {
+  font-size: 2rem;
+  text-transform: uppercase;
+  line-height: 1;
+  font-weight: 400;
+}
+
+#intro > header > p {
+  font-size: 1.1rem;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
+  font-weight: 100;
+  text-transform: none;
+}
+
+#intro > section {
+  padding: 0;
+}
+
+#intro h2 {
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  font-weight: 100;
+  padding-bottom: 2rem;
+}
+
+#intro p {
+  line-height: 1.5rem;
```

Comment:
Now each `<p>` tag will have this line height. Decide if you need class for specific one or it's ok to set this line height for all of them